targets/nuttx-stm32f4/README.md

```diff
@@ -86,6 +86,25 @@ Followings are the options to set:
 * Enable `Application Configuration -> System Libraries and NSH Add-ons -> IoT.js`
 * Enable all childs of `Application Configuration -> System Libraries and NSH Add-ons -> readline() Support` (for those who wants to use readline)
 
+##### 3.1. For those who want to use `net` module
```

Comment:
In line 84, ``Network Device/PHT Support`` should be ``Network Device/PHY Support``.
Could you fix it with this commit?